src/web_interface/templates/analysis_plugins/general_information.html

```diff
@@ -9,39 +9,55 @@
         </colgroup>
         <thead class="thead-light">
             <tr>
-                <th colspan=2> Showing Analysis: {{ selected_analysis | replace_underscore}}</th>
+                <th colspan="2">
+                    <div class="d-flex justify-content-between align-items-center">
+                        <span>
+                            Showing Analysis: {{ selected_analysis | replace_underscore }}
+                        </span>
+                        <button class="btn btn-secondary btn-sm mb-0" onclick="toggleRawResult();">
```

Comment:
This button breaks the html layout in some cases. Or some other hidden element it reveals

When there is, e.g., a raw result for crypto material, the table expands to the right as far as the browser allows.

![20241022_11h31m51s_grim](https://github.com/user-attachments/assets/ee3baba3-0281-4485-9b39-44707784397d)

Reply:
fixed